app/views/users/show.html.erb

```diff
@@ -0,0 +1 @@
+<%= @user.name %>, <%= @user.email %>
```

Comment:
@shaniajubelag
Let's not include this yet, since this is not related to this task.